features/get-computed-style.yml

```
@@ -0,0 +1,8 @@
+name: getComputedStyle()
+description: The `getComputedStyle()` method returns the values of all CSS properties applied to an element.
```

As written, I think it implies that it returns something like an array of values. What do you think of something like this?

```suggestion
description: The `getComputedStyle()` global method returns an `CSSStyleDeclaration` object that represents all CSS declarations applied to a given element.
```